webpack/admin.config.js

```diff
@@ -17,10 +18,24 @@ module.exports = {
     },
     output: {
         filename: 'admin/[name].js',
-        path: __dirname + '/../build',
+        path: `${__dirname}/../build`,
         publicPath: '/',
     },
-    plugins: plugins('admin'),
+    plugins: [
+        definePlugin(),
+        extractTextPlugin('admin'),
+        new HtmlWebpackPlugin({
+            filename: 'admin/index.html',
+            template: `${__dirname}/../src/admin/index.html`,
+            chunks: ['index'],
+            hash: true,
```

Comment:
This is still mentioned on the documentation:
https://github.com/ampedandwired/html-webpack-plugin#configuration